sdk/go/common/resource/plugin/langruntime_plugin.go

```diff
@@ -59,6 +59,11 @@ func NewLanguageRuntime(host Host, ctx *Context, runtime string,
 	for k, v := range options {
 		args = append(args, fmt.Sprintf("-%s=%v", k, v))
 	}
+
+	if project.ConfigFileLocation != "" && runtime == "python" {
```

Comment:
It's a bit unfortunate to have to do anything language specific at this level.  Should we always pass this through to language hots?

Also - should we pull this up to add as `options` to callers instead of handling it separately here?  It feels like this could be injected into language host options a couple levels higher than this?

Reply:
Hmm, yes need another pass tomorrow.

I'll have a closer look at the callers. 

Notes to self: looking again, it seems that `options` are copied into `plugin.NewContext` from `projinfo.Proj.Runtime.Options()`. And `projinfo.Proj` comes straight out the YAML parser.  `NewContext` has call sites in the following places, some of the call sites set nil `options`:

```
pkg/cmd/pulumi/import.go
pkg/cmd/pulumi/policy_publish.go
pkg/codegen/hcl2/binder.go
pkg/codegen/schema/schema.go
pkg/engine/deployment.go
sdk/go/common/resource/plugin/context.go
```

Need to try to simplify this.

Reply:
I've now removed `runtime == "python"` check at the cost of modifying all `pulumi-langage-*` main funcs to accept a new parameter. These programs fail if the parameter is not declared when invoked with the parameter. Only the python runtime actually uses the parameter value at the moment. 

I've opted not to send the new parameter to all options upstream, because it feels like it might require similar edits to resource plugins which becomes a bigger change.